packages/vite-config/src/ConfigBuilder.ts

```diff
@@ -5,15 +5,14 @@ import viteLibraryConfig from './vite-library-config.js'
 import vitestConfig from './vitest-config.js'
 
 export class ConfigBuilder {
-  private includeReactConfig = false
```

Comment:
This can be captured in the existing `pluginConfigOptions` instance variable, so I removed it.